### PR TITLE
Update instructions for current SAML SSO configuration

### PR DESCRIPTION
**Pull request by ari-gold**

### CODE REVIEW

**Comment by ari-gold** [comment]
@ronan Does this look good to you?

File: source/docs/articles/organizations/sso.md [hunk 1]
```diff
@@ -27,29 +26,38 @@ Refer to your IdP for general SAML 2.0 setup instructions.
...
+  * The SAML Request Binding (sent to the IdP from Auth0): `HTTP-Redirect`
+  * The SAML Response Binding (how the SAML token is received by Auth0 from IdP): `HTTP-Post`
+  * The NameID format: `unspecified`
+  * The SAML assertion, and the SAML response can be individually or simultaneously signed.
+  * Optional: Assertions can be encrypted with the following keys: [CER](https://pantheon.auth0.com/cer) | [PEM](https://pantheon.auth0.com/pem) | [PKCS#7](https://pantheon.auth0.com/pb7)
+
+
+<!--If your IdP is Okta or OneLogin, see:
 
 * [Configuring SAML for Pantheon with Okta](/docs/articles/organizations/sso)
```
Comment (ronan) [inline source/docs/articles/organizations/sso.md]: Does this just link back to this same page?

Comment (ari-gold) [inline source/docs/articles/organizations/sso.md]: Yeah, that's in a hidden comment right now tho <!--  ->

**Comment by ronan** [comment]
Ah! I missed the comment in the line above. My bad.

:+1:

**Comment by ari-gold** [comment]
@newtoid Please review

**Comment by ari-gold** [comment]
@nataliejeremy Please review/edit and merge unless @bmackinney has any feedback first.

**Comment by bmackinney** [comment]
:+1:

**Pull request merged**